Model/Payment/Gateway/Service.php

```diff
@@ -19,6 +19,7 @@ class Service
 {
     public const CODE = 'bold';
     public const CODE_FASTLANE = 'bold_fastlane';
+    public const CODE_WALLET= 'bold_wallet';
```

Comment:
Nitpick: You need a space between the constant and the equals sign

Reply:
fixed